### PR TITLE
HLT menu development for `14_0_X` (7/N) [`14_1_X`]

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Development of HLT menus for CMSSW_14_0_X (branch CMSSW_14_1_X).
   * This is the PR for the master branch, which will then be backported down to 14_0_X (the latter is the release cycle currently used for Run-3 HLT-menu development).
   * This version of the GRun HLT menu corresponds to the list of updates to the **"V1.2"** menu for the 2024 pp data-taking period.

This update includes the integration of the following JIRA tickets:
- [CMSHLT-3079](https://its.cern.ch/jira/browse/CMSHLT-3079) : [MUO] Removal pixel hits requirement in Outside-In
- [CMSHLT-3162](https://its.cern.ch/jira/browse/CMSHLT-3162) : [L1T] uGT Test crate data in the NanoDST stream
- [CMSHLT-3169](https://its.cern.ch/jira/browse/CMSHLT-3169) : [HIG] MVA-Based Diphoton HLT Path
- [CMSHLT-3171](https://its.cern.ch/jira/browse/CMSHLT-3171) : [BTV] Migration of emu+1btag to ParticleNet
- [CMSHLT-3172](https://its.cern.ch/jira/browse/CMSHLT-3172) : [L1T] NanoDST stream: store RAW data (`hltFEDSelectorL1`) instead of digis (`hltGtStage2Digis`)
- [CMSHLT-3173](https://its.cern.ch/jira/browse/CMSHLT-3173) : [EXO,Scouting] `AXOL1TL` Paths with `VTight`, `Loose` and `VLoose` Thresholds
- [CMSHLT-3175](https://its.cern.ch/jira/browse/CMSHLT-3175) : [BPH,Scouting,EXO] Scouting ZeroBias (`DST_PFScouting_ZeroBias`)
- [CMSHLT-3176](https://its.cern.ch/jira/browse/CMSHLT-3176) : [BPH,Scouting] update list of L1 bits for scouting muon path
- [CMSHLT-3178](https://its.cern.ch/jira/browse/CMSHLT-3178) : [EXO] Set PS=0 for `HLT_HT360_DisplacedDijet40_Inclusive1PtrkShortSig5_v`
- [CMSHLT-3179](https://its.cern.ch/jira/browse/CMSHLT-3179) : [MUO,Scouting] Update of the NoVtx reconstruction to include extended ROIs (but no pixel doublet recovery)
- [CMSHLT-3180](https://its.cern.ch/jira/browse/CMSHLT-3180) : [BPH] Paths for new low pT BMTF bits
- [CMSHLT-3182](https://its.cern.ch/jira/browse/CMSHLT-3182) : [EGM,EXO,HIG] Reducing the threshold of the high rate single photon path
- [CMSHLT-3183](https://its.cern.ch/jira/browse/CMSHLT-3183) : [EXO] Enabling of AXOL1TL L1 Seeds
- [CMSHLT-3184](https://its.cern.ch/jira/browse/CMSHLT-3184) : [TSG] Backport of online updates in EventDisplay dataset for Cosmics and Circulating menus
- [CMSHLT-3185](https://its.cern.ch/jira/browse/CMSHLT-3185) : [TSG] Prescales of random-based HLTs of GRun in Emergency column (2024 rampup)
- [CMSHLT-3186](https://its.cern.ch/jira/browse/CMSHLT-3186) : [TSG] Add Status_On{CPU,GPU} Paths to all online special menus
- [CMSHLT-3191](https://its.cern.ch/jira/browse/CMSHLT-3191) : [BTV] Adding PNet Info to HLTMonitor
- [CMSHLT-3193](https://its.cern.ch/jira/browse/CMSHLT-3193) : [TSG] Add ZeroBias/HLTPhysics PS columns (Run2024D)
- [CMSHLT-3194](https://its.cern.ch/jira/browse/CMSHLT-3194) : [BTV] Set PS=0 for `HLT_Mu8_*_Ele23_*_CaloDiJet30`

In addition, the following changes are included:
   * d06a530baa046e6fdeb0abc6bf94e647e479b962: update of L1T FULL,GRun,2024v11,Special menus in TSG GTs to `L1Menu_Collisions2024_v1_2_0_xml`
   * f3df2a4f80d8905010ba883499f0e8e587e18b8c : update the list of HLT path owners: menu 2024 V1.2
   * b7613df19710c866259c89f3c07c43ce4d301a5b : Introduce 2024v1.2 frozen menu and remove 2024v1.1

#### PR validation:

TSG Integration tests

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport, but it will be backported to CMSSW_14_0_X.

### CODE REVIEW

**Comment by mmusich** [comment]
type hlt-integration

**Comment by mmusich** [comment]
enable gpu

**Comment by mmusich** [comment]
@cmsbuild, please test

File: HLTrigger/Configuration/tables/Special.txt [hunk 1]
```diff
@@ -95,8 +95,6 @@ HLT_ZeroBias_Gated_v*		    #       CMSHLT-2860
 HLT_SpecialHLTPhysics_v*	    #       CMSHLT-2860
 HLT_SpecialZeroBias_v*		    #       CMSHLT-2860
 
-HLT_L1SingleEG10er2p5_v*	    # CMSHLT-2860
-HLT_L1SingleEG15er2p5_v*	    # CMSHLT-2860
```
Comment (missirol) [inline HLTrigger/Configuration/tables/Special.txt:99]: Mh, I think these 2 are in the (online) Splashes menu, so they should remain in `Special` ? (assuming online-Splashes is derived from Special)

Comment (mmusich) [reply HLTrigger/Configuration/tables/Special.txt:99]: @Martin-Grunewald, you dealt with https://its.cern.ch/jira/browse/CMSHLT-3184, please comment.

Comment (Martin-Grunewald) [reply HLTrigger/Configuration/tables/Special.txt:99]: They are still in the txt file - they had been in there twice, so I removed the duplicates!

Comment (missirol) [reply HLTrigger/Configuration/tables/Special.txt:99]: Okay. Sorry for the noise !

Comment (mmusich) [reply HLTrigger/Configuration/tables/Special.txt:99]: Thanks! I guess this settles it @missirol

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
+hlt
   * changes in the outputs of PR tests are in line with updates to the HLT GRun menu, and the change in name of the frozen HLT menus

**Comment by antoniovilela** [comment]
+1
   - HLT menu V1.2 dump.

**Pull request merged**